### DISCUSSION TITLE
how to speed up installation

### DISCUSSION THREAD

**Discussion by horoyoii** [pyenv/pyenv#2617] [Q&A]
Is there any option like `make -j 20` for speeding up  the installation when I run a `pyenv install` command for python?

**Comment by ajgringo619** [comment]
I have `MAKEFLAGS=-j12` set beforehand, and it seems to help.

**Comment by native-api** [comment]
[It only has a positive effect to increase `-j` to your number of CPU threads for a CPU-bound task like compiling](https://stackoverflow.com/questions/2499070/gnu-make-should-the-number-of-jobs-equal-the-number-of-cpu-cores-in-a-system).

[Which Pyenv already does unless you have set custom `MAKE_OPTS`](https://github.com/pyenv/pyenv/blob/c8daaa3954a607c135bb93fc5890ba6e3a22e1da/plugins/python-build/bin/python-build#L784).

So you don't need to do anything.
